src/config/environments/staging.js

```diff
@@ -34,7 +34,7 @@ exports.data = {
     CRYPTO_SECRET2: process.env.CRYPTO_SECRET2,
   },
   logger: {
-    level: 4,
+    level: 'info',
```

Comment:
Keep this at 4 (or equivalent label for the deeper logger level), as part of our technical debt here is related to the fact that the environment is called 'staging' but in fact is production